docs/quickstarts/fastify.mdx

```diff
@@ -27,171 +64,166 @@ After following this guide, you should have a working Fastify app with public an
 
-  ```env {{ filename: '.env.local' }}
+    The final result should resemble the following:
+  </SignedOut>
+
+  ```env {{ filename: '.env' }}
   CLERK_PUBLISHABLE_KEY={{pub_key}}
   CLERK_SECRET_KEY={{secret}}
   ```
 
-  This examples uses `dotenv` to load the environment variables. You can use any other library or method, if you so wish.
-
-  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
-    ```bash {{ filename: 'terminal' }}
-    npm install dotenv
-    npm install -D @types/dotenv
-    ```
-
-    ```bash {{ filename: 'terminal' }}
-    yarn add dotenv
-    yarn add -D @types/dotenv
-    ```
+  ### Configure `clerkPlugin` for all routes
 
-    ```bash {{ filename: 'terminal' }}
-    pnpm add dotenv
-    pnpm add -D @types/dotenv
-    ```
-  </CodeBlockTabs>
+  The `clerkPlugin` is a Fastify plugin provided by Clerk to integrate authentication into your Fastify application. To ensure that Clerk's authentication and user management features are applied across your Fastify application, configure the `clerkPlugin` to handle all routes or limit it to specific ones.
 
-  ### Configure `clerkPlugin` in your Fastify application
+  The following example registers the plugin for all routes. To register the plugin for specific routes only, refer to the [**Using Clerk for specific pages only**](/docs/quickstarts/fastify#using-clerk-for-specific-routes-only) section.
 
-  The Clerk plugin can be registered globally or for specific routes. This examples registers the plugin globally.
+  > [!IMPORTANT]
+  > The `dotenv/config` module must be imported before any Clerk modules. This order is important because Clerk instances are created during the import process and rely on environment variables, such as API keys, to be initialized correctly. For more details, refer to the [Fastify docs](https://fastify.dev/docs/latest/Guides/Getting-Started/#loading-order-of-your-plugins).
 
   ```ts {{ filename: 'index.ts' }}
-  import * as dotenv from 'dotenv'
-
-  dotenv.config()
-
+  import 'dotenv/config'
   import Fastify from 'fastify'
-  import { clerkClient, clerkPlugin, getAuth } from '@clerk/fastify'
+  import { clerkPlugin } from '@clerk/fastify'
 
   const fastify = Fastify({ logger: true })
 
   fastify.register(clerkPlugin)
 
   const start = async () => {
     try {
-      await fastify.listen({ port: 3000 })
-    } catch (err) {
-      fastify.log.error(err)
+      await fastify.listen({ port: 8080 })
+    } catch (error) {
+      fastify.log.error(error)
       process.exit(1)
     }
   }
 
   start()
   ```
 
-  ### Accessing auth state using `getAuth`
+  ### Use `getAuth()` to access the auth state and protect routes
 
-  The `getAuth` helper can be used to access the auth state of the current request.
+  The `getAuth()` helper retrieves the authentication state from the current request. In the following example, the `userId` loads the entire [`User`](/docs/references/javascript/user/user) object from Clerk.
```

Comment:
```suggestion
  The `getAuth()` helper retrieves the authentication state from the current request.
```

Reply:
let's just keep it consistent across docs. we don't want to say "in the following example" just yet because its repeated two paragraphs below.